master/plan

```diff
@@ -0,0 +1,87 @@
+KINDS:
```

Comment:
note: renaming, potentially removing this for the final merge

Reply:
definitely remove this